go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/tianon/gosu
 
-go 1.16
+go 1.17
+
+require github.com/opencontainers/runc v1.1.0
 
 require (
```

Comment:
This split is interesting -- is this something the `go` tooling did automatically?

Reply:
it seems native in Golang 1.17 : https://go.dev/doc/go1.17#tools